frontend/public/components/machine-set.tsx

```diff
@@ -255,103 +287,209 @@ const MachineSetDetails: React.SFC<MachineSetDetailsProps> = ({ obj }) => {
+        props: { className: tableColumnInfo[4].className },
+        id: tableColumnInfo[4].id,
+      },
+      {
+        title: t('public~Memory'),
+        sort: (data, direction) => data.sort(sortResourceByValue(direction, memory)),
+        transforms: [sortable],
+        props: { className: tableColumnInfo[5].className },
+        id: tableColumnInfo[5].id,
       },
       {
         title: '',
-        props: { className: tableColumnClasses[4] },
+        props: { className: tableColumnInfo[6].className },
+        id: tableColumnInfo[6].id,
       },
-    ];
-  };
+    ],
+    [memory, numCores, t],
+  );
 
-  const MachineSetTableRow: React.FC<RowFunctionArgs<MachineSetKind>> = ({ obj }) => (
-    <>
-      <TableData className={tableColumnClasses[0]}>
-        <ResourceLink
-          kind={machineSetReference}
-          name={obj.metadata.name}
-          namespace={obj.metadata.namespace}
-        />
-      </TableData>
-      <TableData
-        className={classNames(tableColumnClasses[1], 'co-break-word')}
-        columnID="namespace"
-      >
-        <ResourceLink kind="Namespace" name={obj.metadata.namespace} />
-      </TableData>
-      <TableData className={tableColumnClasses[2]}>
-        <Link
-          to={`${resourcePath(
-            machineSetReference,
-            obj.metadata.name,
-            obj.metadata.namespace,
-          )}/machines`}
+  const MachineSetTableRow: React.FC<RowProps<MachineSetKind>> = ({ obj }) => {
+    return (
+      <>
+        <TableData {...tableColumnInfo[0]}>
+          <ResourceLink
+            kind={machineSetReference}
+            name={obj.metadata.name}
+            namespace={obj.metadata.namespace}
+          />
+        </TableData>
+        <TableData
+          {...tableColumnInfo[1]}
+          className={classNames(tableColumnInfo[1].className, 'co-break-word')}
         >
-          {t('public~{{numReadyReplicas}} of {{numDesiredReplicas}} machine', {
-            numReadyReplicas: getReadyReplicas(obj),
-            numDesiredReplicas: getDesiredReplicas(obj),
-            count: getDesiredReplicas(obj),
+          <ResourceLink kind="Namespace" name={obj.metadata.namespace} />
+        </TableData>
+        <TableData {...tableColumnInfo[2]}>
+          <Link
+            to={`${resourcePath(
+              machineSetReference,
+              obj.metadata.name,
+              obj.metadata.namespace,
+            )}/machines`}
+          >
+            {t('public~{{numReadyReplicas}} of {{numDesiredReplicas}} machine', {
+              numReadyReplicas: getReadyReplicas(obj),
+              numDesiredReplicas: getDesiredReplicas(obj),
+              count: getDesiredReplicas(obj),
+            })}
+          </Link>
+        </TableData>
+        <TableData {...tableColumnInfo[3]}>{getMachineSetInstanceType(obj) || '-'}</TableData>
+        <TableData {...tableColumnInfo[4]}>
+          {t('public~{{numCores}} cores', {
```

Comment:
Should this be ...{{count}} core', {count: numCores(obj)} so we catch 1 core 3 cores?

Reply:
Good point! Thanks.